### PR TITLE
Add email for kit recovery

### PR DESCRIPTION
**Pull request by RealOrangeOne**
## Summary

Send an email to the teams who have outstanding kits, asking for them back.

## Checklist

- [x] Has front-matter (`to` and `subject`)
- [x] Compared to a similar email from the previous year
- [x] Ensured there's enough context for a rookie team to understand

### CODE REVIEW

File: SR2024/2024-04-04-kit-recovery.md
Comment (PeterJCLaw) [inline SR2024/2024-04-04-kit-recovery.md:11]: The link texts here aren't super useful as descriptions of what they link to. Best practise for links is that the link text should be as descriptive as possible, so perhaps we could improve this? This is especially important for any users of assistive technologies which may use the link text to indicate which one they want to visit.

In this case there are two links which go to different places yet have the same link text!

```suggestion
Please securely pack the kit and ship it to our [postal address](https://studentrobotics.org/contact/). You can find a [list of the parts](https://studentrobotics.org/docs/kit/) to return on our website or as a checklist attached to this email.
```

**Review by PeterJCLaw — APPROVED** [review]
LGTM, not sure if you want review from the kit team before sending?

File: SR2024/2024-05-04-partial-kit-recovery.md
Comment (PeterJCLaw) [inline SR2024/2024-05-04-partial-kit-recovery.md:3]: Maybe include the year here? (also in the other email)
```suggestion
subject: Outstanding kit from SR2024
```

**Review by raccube — APPROVED** [review]

**Pull request merged**
